docs/ember/template-registry.md

```diff
@@ -2,12 +2,16 @@ Because Ember's template resolution occurs dynamically at runtime today, Glint n
 
 The recommended approach is to include a declaration in each component, modifier or helper module that adds it to the registry, which is the default export of `@glint/environment-ember-loose/registry`.
```

Comment:
This section could use an example of what the glint error message will look like if you forget to add something to the registry so that Google will point someone to this documentation when that happens.